crates/containerd-shim-wasmedge/src/instance.rs

```diff
@@ -136,29 +131,6 @@ impl LibcontainerInstance for Wasi {
     }
 }
 
-impl EngineGetter for Wasi {
```

Comment:
Where is this code moved to?

Reply:
This wasn't actually used, so it's just removed.